pandas/tests/io/pytables/test_store.py

```diff
@@ -784,7 +784,7 @@ def test_complibs(self, setup_path):
                 gname = "foo"
 
                 # Write and read file to see if data is consistent
-                df.to_hdf(tmpfile, gname, complib=lib, complevel=lvl)
+                df.to_hdf(tmpfile, gname, complib=lib, complevel=lvl, format="table")
```

Comment:
Can you add a dedicated test for the change you are making instead of just modifying this?